Strange, it didn't help, error is still here:
```
Executing subproject pcre2_cross_native 

pcre2_cross_native| Project name: pcre2_cross_native
pcre2_cross_native| Project version: 10.43
pcre2_cross_native| C compiler for the host machine: /usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android33-clang (clang 14.0.7 "Android (9352603, based on r450784d1) clang version 14.0.7 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0cca074e9238af8b4106c30add4418f6)")
pcre2_cross_native| C linker for the host machine: /usr/local/lib/android/sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android33-clang ld.lld 14.0.7
pcre2_cross_native| C compiler for the build machine: cc (gcc 11.4.0 "cc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0")
pcre2_cross_native| C linker for the build machine: cc ld.bfd 2.38

subprojects/pcre2_cross_native-10.43/meson.build:11:19: ERROR: File src/pcre2_chartables.c.dist does not exist.

A full log can be found at /home/runner/work/rizin/rizin/build/meson-logs/meson-log.txt
Error: Process completed with exit code 1.
```

--- a/subprojects/packagefiles/pcre2/meson.build
+++ b/subprojects/packagefiles/pcre2/meson.build
@@ -1,4 +1,4 @@
-project('pcre2', 'c', version: '10.42')
+project('pcre2', 'c', version: '10.43')
 
 cc = meson.get_compiler('c')
 
@@ -28,6 +28,7 @@ libpcre2_c_args = [
 pcre2_files = [
   'src/pcre2_auto_possess.c',
   pcre2_chartables,
+  'src/pcre2_chkdint.c',
   'src/pcre2_compile.c',
   'src/pcre2_config.c',
   'src/pcre2_context.c',

--- a/subprojects/packagefiles/pcre2_cross_native/meson.build
+++ b/subprojects/packagefiles/pcre2_cross_native/meson.build
@@ -2,7 +2,7 @@
 # Because Meson currently doesn't support subprojects to be native and non-native:
 # https://github.com/mesonbuild/meson/issues/10947
 
-project('pcre2_cross_native', 'c', version: '10.42')
+project('pcre2_cross_native', 'c', version: '10.43')
 
 cc = meson.get_compiler('c')
 
@@ -32,6 +32,7 @@ libpcre2_c_args = [
 pcre2_files = [
   'src/pcre2_auto_possess.c',
   pcre2_chartables,
+  'src/pcre2_chkdint.c',
   'src/pcre2_compile.c',
   'src/pcre2_config.c',
   'src/pcre2_context.c',

--- a/subprojects/pcre2.wrap
+++ b/subprojects/pcre2.wrap
@@ -1,7 +1,8 @@
-[wrap-git]
-url = https://github.com/PCRE2Project/pcre2.git
-revision = 52c08847921a324c804cabf2814549f50bce1265
-directory = pcre2
+[wrap-file]
+source_url = https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.43/pcre2-10.43.tar.gz
+source_filename = pcre2-10.43.tar.gz
+source_hash = 889d16be5abb8d05400b33c25e151638b8d4bac0e2d9c76e9d6923118ae8a34e
+directory = pcre2-10.43
 patch_directory = pcre2
 
 [provide]

--- a/subprojects/pcre2_cross_native.wrap
+++ b/subprojects/pcre2_cross_native.wrap
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/PCRE2Project/pcre2.git
-revision = 52c08847921a324c804cabf2814549f50bce1265
+revision = 3864abdb713f78831dd12d898ab31bbb0fa630b6
 directory = pcre2_cross_native
 patch_directory = pcre2_cross_native
 

